bridges/PanneauPocketBridge.php

```diff
@@ -0,0 +1,167 @@
+<?php
+class PanneauPocketBridge extends BridgeAbstract {
+	const NAME = 'Panneau Pocket';
+	const URI = 'https://app.panneaupocket.com';
+	const DESCRIPTION = 'Fetches the latest infos from Panneau Pocket';
+	const MAINTAINER = 'floviolleau';
+	const PARAMETERS = array(array(
+				'cities' => array(
+					'name' => 'Choisir une ville',
+					'type' => 'list',
+					'values' => self::CITIES
+				 )
+			   ));
+	const CACHE_TIMEOUT = 7200;
+
+	// debug function to get cities
+	private function getCities() {
+		$url = self::URI . '/public-api/city';
+		$ch = curl_init();
+		curl_setopt($ch, CURLOPT_URL, $url);
+		curl_setopt($ch, CURLOPT_USERAGENT, 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:80.0) Gecko/20100101 Firefox/80.0');
+		curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+		curl_setopt($ch, CURLOPT_FOLLOWLOCATION, 1);
+		curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, 0);
+		$content = curl_exec($ch);
```

Comment:
Do not use curl_exec. Use `getContents` with custom `CURL_OPTS` set instead.
Example:
```
$curl_opts = [CURLOPT_FOLLOWLOCATION => 1];
$content = getContents($url, [], $curl_opts);
```